client/index.js

```diff
@@ -1,22 +1,66 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import ReactDOM from 'react-dom';
 import Map from './Components/Map';
 import NearbyTransitList from './Components/NearbyTransitList';
 
-const App = () => (
-  <div className="map-wrapper">
-    <h2>Location</h2>
-    <address style={{
-      'whiteSpace': 'pre-line',
-      'marginBottom': '2rem'
-  }}>
-    415 Mission Street<br />
-    San Francisco, CA 94105
-    </address>
-    <Map />
-    <NearbyTransitList transitOptions={['option 1', 'option 2', 'option 3']}/>
-  </div>
-)
+const App = () => {
+
+  const [locationData, updateLocationData] = useState(
+    {
+      coordinates: [0, 0],
+      streetName: 'loading',
+      streetNumber: 'loading',
+      city: 'loading',
+      state: 'loading',
+      zip: 'loading'
+    });
+
+  const [nearbyTransits, updateNearbyTransits] = useState([]);
+
+  const { streetName, streetNumber, city, state, zip } = locationData;
+
+  useEffect(() => {
+
+    const id = window.location.pathname.split('/')[2];
```

Comment:
Safer would be to get last element from location, just in case the URL changes length. Emmanuel used this solution:
```
// Filter to remove empty elements, in case the URL has a trailing slash
let splitUrl = window.location.pathname.split('/').filter(el => el);
    let rawId =  splitUrl[splitUrl.length - 1];
```

Reply:
updated